docs/reference/admin/databases.rst

```diff
@@ -50,6 +50,16 @@ Remove a database.
 
     drop database <name> ;
 
+.. versionadded:: 5.0
+
+    You may now drop a branch, even if it has active connections by adding
+    ``force``:
```

Comment:
Do we need some kind of a warning here to say, this is not usually recommended because it may interrupt ongoing transactions, use with extra caution? @msullivan